### PR TITLE
Adding DualStack tips

### PR DESCRIPTION
**Pull request by ptrnull**
OpenShift IPv4/IPv6 dualstack tips

### CODE REVIEW

File: content/networking/_index.md [hunk 1]
```diff
@@ -1,10 +1,124 @@
...
 ---
 
-Nothing to see here (yet)
+# IPv4 IPv6 DualStack
+
+At the time of writting this document the priority IP protocol inside and OpenShift cluster is IPv4, that means that by default the first IP that are getting the nodes, pods and services is IPv4. 
+
+**Installation**
+
+The install-config.yml needs to include both network configurations, IPv4 and IPv6, an example:
```
Comment (e-minguez) [inline content/networking/_index.md:16]:
```suggestion
The `install-config.yaml` file needs to include both network configurations, IPv4 and IPv6, an example:
```

File: content/networking/_index.md [hunk 2]
```diff
@@ -1,10 +1,124 @@
...
+    externalBridge: ostestbm
+    bootstrapOSImage: http://192.168.111.1/images/rhcos-47.83.202101161239-0-qemu.x86_64.qcow2.gz?sha256=3a14ff77b4b7a5c89d145226759c71e852bd54eb8eea50866e760c801c7b623a
+    clusterOSImage: http://192.168.111.1/images/rhcos-47.83.202101161239-0-openstack.x86_64.qcow2.gz?sha256=ccc2c776ce3d4bbb7585fdf497286c3694633f609bf7aeee42c5f8c274560bd2
+    apiVIP: 192.168.111.5
+    ingressVIP: 192.168.111.4
+```
+
+Pay special attention to the order of the networks, first IPv4, second IPv6, at the moment that is important and mandatory.
+
+The _FeatureGate_ also needs to be enabled:
```
Comment (e-minguez) [inline content/networking/_index.md:48]:
```suggestion
The `IPv6DualStackNoUpgrade` _FeatureGate_ also needs to be enabled by adding the following to the list of manifests:
```

File: content/networking/_index.md [hunk 3]
```diff
@@ -1,10 +1,124 @@
...
+metadata:
+  name: cluster
+spec:
+  featureSet: IPv6DualStackNoUpgrade
+
+```
+
+**Setting up IPv4, IPv6 or DualStack in a service**
+
+After the cluster is up and running you can test the IPv6 configuration with a Service. By default the Services are created IPv4 only, if you want to create a Service with IPv6 you have to pay attention to the following _ipFamilies_ and _ipFamilyPolicy_ sections:
```
Comment (e-minguez) [inline content/networking/_index.md:62]:
```suggestion
After the cluster is up and running you can test the IPv6 configuration with a Service. By default the Services are created IPv4 only, if you want to create a Service with IPv6 you have to modify the `spec.ipFamilyPolicy` setting.

The default setting is to expose only as a `SingleStack` cluster IP for the Service, using the first configured service cluster IP range (IPv4 by default):
```

File: content/networking/_index.md [hunk 4]
```diff
@@ -1,10 +1,124 @@
...
+  - name: 8888-tcp
+    port: 8888
+    protocol: TCP
+    targetPort: 8888
+  selector:
+    deployment: hello-openshift
+  sessionAffinity: None
+  type: ClusterIP
+```
+For the case of using DualStack:
```
Comment (e-minguez) [inline content/networking/_index.md:92]:
```suggestion
In order to use DualStack, the `spec.ipFamilyPolicy` setting needs to be configured whether to `PreferDualStack` or `RequireDualStack` as:
```

File: content/networking/_index.md [hunk 5]
```diff
@@ -1,10 +1,124 @@
...
+    port: 8888
+    protocol: TCP
+    targetPort: 8888
+  selector:
+    deployment: hello-openshift
+  sessionAffinity: None
+  type: ClusterIP
+```
+
+For more information you can check the upstream documentation: https://kubernetes.io/docs/concepts/services-networking/dual-stack/#forum-app-services
```
Comment (e-minguez) [inline content/networking/_index.md:123]:
```suggestion
For more information about IPv4/IPv6 dual stack you can check the [upstream documentation](https://kubernetes.io/docs/concepts/services-networking/dual-stack/)
```

**Review by e-minguez — CHANGES_REQUESTED** [review]
A few comments/nits :) Thanks!

**Review by ptrnull — COMMENTED** [review]
applied, thanks!

**Comment by e-minguez** [comment]
Merged, thanks! 🚀

**Pull request merged**